app/build.gradle

```diff
@@ -308,6 +313,9 @@ android {
     }
 
     compileOptions {
+        // Flag to enable support for API desugaring
+        coreLibraryDesugaringEnabled true
```

Comment:
should be another PR as well ideally as it's not related to CT at all

Reply:
I am actually going to split this out in another PR as we need it in `master` right away.